### PR TITLE
Branch UG improvements

### PR DESCRIPTION
**Pull request by zylee348**
Added GUI interface and about UG command formats and prefixes

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -1,50 +1,51 @@
...
-![TABS example](images/user-guide/Ui.png)
-<br><br>
-5. Refer to the [Features](#features) below for details of each command.
+TABS is a desktop application which helps student project team leaders **overview tasks assigned to group members**.
+TABS helps in 3 key ways:
+1. Contact details management 
+2. Group management
+3. Delegated task management
 
+TABS aim to reduce the amount of time spent on managing team projects for **students** who already have their academics to contend with. TABS is for people who type quickly as it **optimised for use via Command Line Interface** (CLI) while still having the benefits of a Graphical User Interface (GUI). TABS is available for the **Linux, Windows and Mac OS** operating systems.
```
Comment (Tan-Jin-Waye) [inline docs/UserGuide.md:14]: aim should be aims.

Comment (zylee348) [reply docs/UserGuide.md:14]: thankstahks

**Review by LianGuoYang — APPROVED** [review]
LGTM

**Comment by CedricChia123** [comment]
LGTM

**Pull request merged**